### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by danclery**
Here's an attempt to spruce up the Welcome page. Sorry this has taken so long! In the "primary goals" section, I removed a few items which seemed important to developers but less so to users. But it's obviously up to you guys. Let me know what you think.

### CODE REVIEW

File: docs/EN/index.rst [hunk 1]
```diff
@@ -3,29 +3,34 @@ Welcome to the Android APS documentation
...
-* app where you can see how APS works: input params, result and final decision
-* ability to add more APS algorithms and let the user decide what to use
-* app which is independent from a pump driver and contains a "Virtual pump" to allow users to safely play with APS
-* app with tight Nightscout integration
-* app where is easy to add/remove constraints for user safety
-* all-in-one app for managing T1D with APS and Nightscout
+* An app with safety built in. To read about the safety features of the algorithms, known as oref0 and oref1, click here (https://openaps.org/reference-design/)
+* An all-in-one app for managing type 1 diabetes with an artificial pancreas and Nightscout
+* An app to which users can easily add or remove modules as needed
+* An app that allows localization {I don't know what this means. Do you mean it can be used offline?}
```
Comment (AdrianLxM) [inline docs/EN/index.rst]: Hi, there some information is put in.
Localization is the step after internationalisation ;) (A bit more than just translations)

File: docs/EN/index.rst [hunk 2]
```diff
@@ -3,29 +3,34 @@ Welcome to the Android APS documentation
...
+* A continuous clucose monitor (CGM): Dexcom G4/G5/G6, Freestyle Libre, Eversense, Medtronic Guardian, or PocTech
+* An app on the phone to receive CGM data: `xDrip <http://stephenblackwasalreadytaken.github.io/xDrip/>`_/ `xDrip+ <https://jamorham.github.io/#xdrip-plus>`_, `Glimp <https://play.google.com/store/apps/details?id=it.ct.glicemia>`_ , `G5 patched app <https://github.com/dexcomapp/dexcomapp>`_, `PochTech app <https://play.google.com/store/apps/details?id=jp.co.unitec.concretemanagement&hl=gsw>`_ or `600SeriesAndroidUploader <http://pazaan.github.io/600SeriesAndroidUploader/>`_
+* `AndroidAPS <https://github.com/MilosKozak/AndroidAPS>`_ itself installed on the phone
 * `Nightscout cgm-remote-monitor <http://www.nightscout.info/wiki/welcome/set-up-nightscout-using-heroku>`_ 0.10.2 or later
-* A supported pump: Dana-R or Dana-RS Insulin Pump or Accu-Chek Combo (unless you build your own driver for another insulin pump)
-* A Continuous Glucose Monitor (CGM) data source: Dexcom G4/G5/G6, Freestyle Libre, Eversense, Medtronic Guardian, PocTech
```
Comment (AdrianLxM) [inline docs/EN/index.rst]: I believe this was deleted?

Comment (danclery) [reply docs/EN/index.rst]: I thought I had move them, rather than deleted them.

File: docs/EN/index.rst (hunk 1 above)
Comment (danclery) [reply docs/EN/index.rst]: Could we say something like: Available in multiple languages. Or is that simplifying too much?

Comment (danclery) [reply docs/EN/index.rst]: How about this: 
An app with different versions for specific locations and languages.

File: docs/EN/index.rst (hunk 2 above)
Comment (danclery) [reply docs/EN/index.rst]: Yes, the CGM line is higher up and the pump line is below.

**Comment by danclery** [comment]
Adrian, I'm not sure how to resolve the conflicts mentioned above. Are you able to do that? Do I need to do anything else on this page?

**Comment by MilosKozak** [comment]
👍

**Comment by typ1-brods** [comment]
@AdrianLxM I can't see any merging conflicts. Do you?

**Comment by MilosKozak** [comment]
i fixed it ;)

**Comment by danclery** [comment]
Thanks! So is it good to go?

**Pull request merged**